### ISSUE TITLE
Reviving Lei Shen

### ISSUE THREAD

**Issue by Gaben1025** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#797]
**Describe your suggestion in full detail below:**
- Add a decision to revive lei shen

**Comment by Remerod** [comment]
This decision needs to be some sort of combined effort between Mogu and Zandalari characters in my opinion. (Note that I'm not aware/remembering how this is done in ck2 at this moment, so forgive my ignorance if this is already done)

To my limited Pandaria lore i know that:
* Lei Shen was told of the revival ritual aeons ago but never told any of his retainer how to do this.
* His Body was buried in the **Tomb of Conquerors** 
* He is resurrected during WoW: Mists of Pandaria by the Zandalari, specifically by Prophet Khar'zul the leader of Zandalari expedition in Pandaria.

So the Decision in my opinion would need a independent Mogu ruler (below at least empire i would assume) holding **Tomb of Conquerors** and somehow ask either a Zandalari expedition on Pandaria or the God King of Zandalar. The Zandalari who is asked most have a **proper priest/prophet** capable of this task. Lei Shen is revived, creates a title above the mogu who owns **Tomb of Conquerors** (perhaps Lei Shen gets the county title Tomb of Conquerors is in). And then i guess Lei Shen would get claims to his storm isles but that's not part of this event.

Thoughts on this?

**Comment by zumbak04** [comment]
> (Note that I'm not aware/remembering how this is done in ck2 at this moment, so forgive my ignorance if this is already done similar there)

Either the Zandalari who remembered about Lei Shen's deal to give the Zandalari a big chunk of Pandaria or mogu rulers could resurrect him.

> The Zandalari who is asked most have a proper priest/prophet capable of this task.

Both the Zandalari and mogu know his resurrection secret, so Lei Shen may be resurrected without the help of the Zandalari.

**Comment by Gaben1025** [comment]
Ya I would do both, also if he were to ever end up as a prisoner to one of his enslaved cultures there should be some extra options probably for punishment

**Comment by Remerod** [comment]
> Both the Zandalari and mogu know his resurrection secret, so Lei Shen may be resurrected without the help of the Zandalari.

I might be wrong, but as i remembered it, Lei Shen kept the secrets of the ritual away from the Mogu, so they did "not" know of this secret, and is the reason why Lei Shen was not resurrected earlier. and this being a thing in Mist of Pandaria because the Zandalari can now get threw the mist and thus do the ritual where it was not possible before the mist.... demisting? (mist stops be a mist because of cata)

Found this: 
WoWpedia Lei Shen/Reign of the Thunder King
> Lei Shen forged an alliance between the mogu and Zandalar tribe of trolls, led by Zulathra, and this alliance would last millennia, long after Lei Shen had passed on. Lei Shen made a promise to them: a large plot of land near the Vale of Eternal Blossoms would forever belong to the trolls. Lei Shen, not trusting his mogu underlings to resurrect him if killed, also told the Zandalari trolls the secret to his resurrection.[6]

WoWpedia Lei Shen/Death
> Years after the Thunder King's death and near the end of the Zandalari Troll Wars, the leader of the invading Zandalari armies, Mengazi - a descendant of Zulathra - realized that the trolls were losing against the pandaren and their cloud serpent riders, and decided to fulfill the ancient prophecy and resurrect Lei Shen. A pitched battle erupted at the Tomb of Conquerors, where the Thunder King's body was enshrined. The trolls' efforts to resurrect Lei Shen were thwarted, as Jiang sacrificed herself to kill Mengazi, sending the Zandalari fleeing back to Zandalar.[6]

So not only was the trolls the only ones beside Lei Shen that knew this, they have also tried to resurect him pre-MoP, but failed.

**Comment by Gaben1025** [comment]
Maybe a decision to ask the Zandalari priest for the technique if the opinion is over a certain level

**Comment by Remerod** [comment]
I dunno, Lei Shen died before the sundering, we are talking about a Legend of Aeons past. I don't think revivng Lei Shen should be easy. i think it definitly should be a event chain in a Mist of Pandaria bookmark! but it should not be the obvious choice pre-that. I personally think the best was would be for the Zandalari to have a claim on the part they take in MoP and when come into contact with each other, they'll be like;
Mogu: what are ya doing in these lands? we where ancient allies before but that was long time ago, these alnds now belong to us
Zandalari: we want alliance and these pandaria lands again, we'll revive your god-king if you agree
Mogu: ok, sounds good

Something like that :P would be ideal to me, I don't think unlocking Lei Shen should be every Mogu's dream, i very much expect foe example Xin the Weaponmaster (current King of the Mogu Clans) to be happy to be the strongest Mogu and not wanting rivalry even if that rival would be the legendary Lei Shen. the Mogu can be loyal, but most Mogu aspire power for themselves, the Mogu revered Lei Shen because he was the most powerful Mogu by miles and wanted to lead the Mogu to be even more powerful. Lei Shen should definitely be a path, but it should not be the main path the Mogu playstile leads to. but Lei Shen as I mentioned earlier should be a thing for it's bookmark, i'm not saying we should not have it before the bookmark. I'm just saying we should not make it to easy unlocking/releasing him. Some stuff are more badass if you have to work for it after all

**Comment by Gaben1025** [comment]
I agree it shouldnt be easy, I disagree that it should be locked to a date bc that bookmark is way to far in the future. When it comes time to make that bookmark, Lei should be alive and available though

**Comment by Gaben1025** [comment]
If I understand there was nothing stopping the Zandalari from doing it earlier.

**Comment by Remerod** [comment]
> I agree it shouldnt be easy, I disagree that it should be locked to a date bc that bookmark is way to far in the future. When it comes time to make that bookmark, Lei should be alive and available though 

I never said that, I was trying to say it should be a natural event/outcome in the MoP bookmark, while harder in the others
I don't oppose him being alive in that bookmark but then we have to change some stuff, like Xin and the mantid empress being alive, since they would have died by the time his resurrection came 

> If I understand there was nothing stopping the Zandalari from doing it earlier.

Expect a massive impenetrable Mist? 

> The mists of Pandaria[1][2] were magical mists conjured by the Emperor of Pandaria, Shaohao. They protected the inhabitants of the lands during the Great Sundering of Kalimdor at the end of the War of the Ancients. 
According to the pandaren Anglers, no boat could find its way home through the mists.[8] Even though the mists were dense and impenetrable,[9] the keen adventurer and explorer Liu Lang on his giant turtle called Shen-zin Su traveled between shrouded Pandaria and the other continents of Azeroth. He gathered like-minded pandaren and never returned to Pandaria again, avoiding it.[10]
Over 10,000 years later, the mists parted due to the effects of Deathwing's Cataclysm.[11] Shortly after the Attack on Theramore Isle, the island continent was then first found by Chen and Li Li Stormstout thanks to the visions granted by the Pearl of Pandaria,[12] soon followed by the forces of the Alliance and Horde.[13][14] Even though they fell,[15] the mists still remained in the sky.

**Comment by zumbak04** [comment]
> Lei Shen kept the secrets of the ritual away from the Mogu, so they did "not" know of this secret

Confirm it. From the Chronicles.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/145372190-0c5b40e3-63a2-4005-b58c-5cff29c6d1c5.png)

</details>
